addons/website/views/website_templates.xml

```diff
@@ -1552,8 +1577,8 @@
 </template>
 
 <template id="header_navbar_pills_style" inherit_id="website.navbar_nav" name="Navbar Links Style" active="False">
-    <xpath expr="//ul[@id='top_menu']" position="attributes">
-        <attribute name="t-attf-class" add="nav-pills" separator=" "/>
+    <xpath expr="//ul[@id='top_menu']" position="before">
+        <t t-set="_navbar_pills" t-value="True"/>
```

Comment:
If this variable manages the "Rounded box" menu, its name will conflict with the "Links Style - Pills" option.
Could you double check if it still works with this option ?

Reply:
Thanks, indeed there was missing some code to manage the link style do be disabled on the new mobile offcanvas and the sidebar + hamburger headers. I did a new commit addressing this issue.